submissions/Mifaresss/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Popup | Homework Kottans</title>
+	<link href="https://fonts.googleapis.com/css?family=Noto+Sans:regular,700&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="./css/nulling-styles.css">
+	<link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+	<header class="header">
+		<div class="wrapper">
+			<div class="header__container">
+				<a href="#" class="header__logo">PopUp</a>
```

Comment:
Fix elements shifting on focus, [watch video](https://share.vidyard.com/watch/S25tSwGfnDSLJnDoaLR1ax?). Extra rerenders are bad for page performance. Elements should not move when one of them is in focus.

Reply:
Ok

Reply:
Is there a need for improvement here?